authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHDFSPlugin.java

```diff
@@ -278,10 +278,10 @@ private void removeSchemaMetadataObject(AuthorizationMetadataObject authzMetadat
         authzMetadataObject instanceof PathBasedMetadataObject,
         "The metadata object must be a PathBasedMetadataObject");
     Preconditions.checkArgument(
-        authzMetadataObject.type() == SCHEMA, "The metadata object type must be SCHEMA");
+        authzMetadataObject.type() == SCHEMA, "The metadata object type must be a schema");
     Preconditions.checkArgument(
         authzMetadataObject.names().size() == 1,
-        "The size of the metadata object's name must be 1.");
+        "The size of the metadata object's size must be 1.");
```

Comment:
Check this again?

Reply:
"name must be 1" makes no sense unless you mean the name must be the letter 1, which is not intended here.

Reply:
I mean "The size of the metadata object's size" doesn't make sense ... WDYT?

Reply:
it makes sense to me and I don't think most peopel would find that confusing - as a Precondition, it's more aimed at developers.